packages/bitcoin-dlc-provider/lib/BitcoinDlcProvider.ts

```diff
@@ -774,7 +774,14 @@ export default class BitcoinDlcProvider extends Provider {
     const hyperbolaCurve = HyperbolaPayoutCurve.fromPayoutCurvePiece(
       hyperbolaPayoutCurvePiece,
     );
-    const payout = hyperbolaCurve.getPayout(outcome);
+
+    const clampBN = (val: BigNumber) =>
```

Comment:
I think you need to import BigNumber as well

Reply:
oof right